docs/learn-scams.md

```diff
@@ -0,0 +1,38 @@
+---
+id: learn-scams
+title: How to Protect Yourself from Scams
+sidebar_label: How to Protect Yourself from Scams
+---
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
+
+### Admin will never contact you over Telegram
+
+If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
+personally message you. Our social media's are posted on our website and any new social media's will
+be announced by our team.
+
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, etc. If something seems fishy, that's because it likely is.
```

Comment:
Many scammers will create nearly perfect imitations of sites - triple-check the URL!  Never enter any personal data after clicking on a link, always type it in yourself.